apps/server/src/shared/repo/task/task.repo.integration.spec.ts

```diff
@@ -860,6 +1120,125 @@ describe('TaskRepo', () => {
 	});
 
 	describe('findAllFinishedByParentIds', () => {
+		describe('with filters', () => {
+			describe('with assigned users filter', () => {
```

Comment:
is 'with assigned users filter' a specific case of ''with filters'? because I dont see another differentiation for 'with filters'